qt/scientific_interfaces/Indirect/IndirectSettings.cpp

```diff
@@ -8,16 +8,28 @@
 #include "IndirectInterface.h"
 #include "IndirectSettingsHelper.h"
 #include "MantidQtIcons/Icon.h"
+#include "MantidQtWidgets/Common/UserSubWindow.h"
 
 constexpr auto SETTINGS_ICON = "mdi.settings";
 
 namespace MantidQt::CustomInterfaces {
-DECLARE_SUBWINDOW(IndirectSettings)
 
-IndirectSettings::IndirectSettings(QWidget *parent) : MantidQt::API::UserSubWindow(parent) { m_uiForm.setupUi(this); }
+IndirectSettings::IndirectSettings(QWidget *parent) : QWidget(parent) {
+  auto model = std::make_unique<IndirectSettingsModel>();
+  m_presenter = std::make_unique<IndirectSettingsPresenter>(std::move(model), new IndirectSettingsView(this));
+  m_presenter->subscribeParent(this);
+
+  auto layout = new QGridLayout();
```

Comment:
Just to confirm, given the the use of `new` here, does Qt handle the garbage collection for these pointers once they've been closed?

Reply:
Good question, just had a read of the docs here and it says that the QWidget takes ownership of the QLayout
https://doc.qt.io/qt-5/qwidget.html#setLayout